README.md

```diff
@@ -13,27 +13,28 @@ We use:
 
 ## Development
 
-In order to debug backend code in browser do several steps:
+### Setup
 
-  `cd server && yarn debug`
+WIP: `yarn setup` script will be implemented for faster development start
 
-  visit _about://inspect_ in Chrome and click *Open dedicated DevTools for Node* link
+1) You may want to install packages for both server and client apps, here you are:
 
-  put `debugger` into your code
+in root dir: `yarn update`
 
-  enjoy
+2) Create database using psql:
```

Comment:
@AleksSenkou we can create db via `node_modules/.bin/sequelize db:create` 😉

Reply:
@DDKatch can we do `NODE_ENV=test node_modules/.bin/sequelize db:create` for testing db creation?

Reply:
@AleksSenkou yep